data/blog/2024-year-in-review.mdx

```diff
@@ -0,0 +1,545 @@
+we think about individual sovereignty and liberty—values that we hold in high
+regard. We want to operate with as little overhead as possible, which is why we
+don't take a cut from donations. And we don't want any one person
+or small group to have control over funding or organizational decisions, which
+is why we have a 9-person board that is the final arbiter of all decisions.
+
+We don't claim to have all the answers, and we don't claim that what we are
+doing is frictionless or perfect. We are bound to make mistakes, and we will
+always be prone to false positives (funding projects that aren't as fruitful as
+we thought they would be) as well as false negatives (not funding projects that
+should receive funding). We strive to gradually and continually improve our
+processes, and we hope to get better at what we are doing with every passing
+year. That said, we are incredibly pleased to see that our structure and
+processes are working, and we believe this year in review is a testament to
+that.
+
+---
+
+## Announcements
+
+At the beginning of the year, we announced long-term support for [René
+Pickhardt](/blog/rene-pickhardt-receives-lts-grant) and
+[0xB10C](/blog/0xB10C-receives-lts-grant), two researchers/developers with a
+major track record of doing important work related to bitcoin infrastructure.
+Soon after we announced long-term support for [Bruno
+Garcia](/blog/bruno-garcia-receives-lts-grant), another developer that qualifies
+as a [load-bearing internet
+person](/blog/announcing-lts-grant-program-to-support-bitcoin-core-contributors)
+by our estimation. In May [Shashwat
+Vangani](/blog/shashwat-vangani-receives-lts-grant) joined our roster of LTS
+grantees, followed by [Calle](/blog/cashu-calle-receives-lts-grant) in June, and
+[Jason Donenfeld](/blog/jason-donenfeld-lts-grant)
+&
+[Jon Atack](/blog/jon-atack-receives-lts-grant)
+in September. The last two long-term
+support grants in 2024 were awarded to [Dusty
+Daemon](/blog/dusty-daemon-receives-lts-grant) and
+[m1sterc001guy](/blog/m1sterc001guy-receives-lts-grant), as announced in October
+and November, respectively. This brings the total number of grantees receiving
+long-term support from our General Fund to 18, adding to the 9 announced in 2023
+([Marco](/blog/announcing-lts-grant-program-to-support-bitcoin-core-contributors),
+[Josi](/blog/josi-baker-receives-opensats-lts-grant),
+[Sjors](/blog/sjors-provoost-receives-opensats-lts-grant),
+[Vasil](/blog/vasil-dimov-receives-lts-grant),
+[Gleb](/blog/gleb-naumenko-receives-lts-grant),
+[Matt](/blog/matt-morehouse-lightning-security-lts-grant),
+[Will](/blog/will-clark-receives-lts-grant-for-bitcoin-core),
+[Furszy](/blog/furszy-receives-lts-grant-for-bitcoin-core), and
+[Tobin](/blog/tobin-harding-receives-lts-grant-for-rust-bitcoin)).
+
+![](/static/images/blog/2024-eoy/lts-grantees.png)
+
+Speaking of long-term support, in March we announced the expansion of our LTS
```

Comment:
"In March we expanded our long-term support program to include nostr developers."

More concise.